views/use_the_data.erb

```diff
@@ -0,0 +1,256 @@
+<div class="container" id="contribute">
+  <div class="page-section">
+    <div class="row">
+      <div class="column-one-quarter">
+         <%= erb :about_nav_menu %>
+      </div>
+      <div class="column-three-quarters">
+        <h1>Use EveryPolitician data</h1>
+
+        <p class="section-intro">
+			We gather together all this lovely data, curate it, and make it
+			reliably available... so you can use it. Here&rsquo;s the bare bones of how
+			your application can get it.
+        </p>
+        <p>
+			This page is for you if you&rsquo;re a developer who is planning on
+			building something wonderful or brilliant with the data we have on your
+			country&rsquo;s politicians. It tells you how to get the data and, broadly,
+			how it&rsquo;s structured (spoiler: CSV or Popolo JSON).
+        </p>
+		<p>
+			We believe that by making this data available in consistent and
+			useful formats, if you write something neat that uses, for example, the data
+			for your country&rsquo;s politicians, and open it up under a free software open
+			source license... then other people will be able to adapt what you&rsquo;ve
+			done to run in their country too. That works because their data will be
+			formatted the same way that yours is: that&rsquo;s our side of the deal.
+		</p>
+		<h2>
+			How to find what data we&rsquo;ve got for you
+		</h2>
+		<p>
+			Humans look at the <a href="/">map on the front page</a>. But
+			you&rsquo;re a developer: you look in
+			<code><a href="https://raw.githubusercontent.com/everypolitician/everypolitician-data/master/countries.json">countries.json</a></code>.
+		</p>
+		<p>
+			That&rsquo;s because <code>countries.json</code> is effectively
+			EveryPolitician&rsquo;s index. It contains the complete set of countries we
+			have data for as a JSON array of country objects. Those contain identifying
+			fields (like name and country code) but also the SHAs of the git commits of
+			the most recent versions of each legislature within those countries, and the
+			paths you need to <em>find</em> them. From <code>countries.json</code>, you
```

Comment:
Clarified by dropping some of the detail (e.g., about the SHAs: that's described in the part where we actually *use* them) so just say "metadata", cos that's what it is --

"These contain identifying fields (like name and country code) as well as metadata for each legislature within those countries. From countries.json you can construct the URLs of the files that contain the data you want."